### ISSUE TITLE
Picture not saved due to 'Could not save image' error

### ISSUE THREAD

**Issue by rsahlin** [mapillary/mapillary_issues#1959]
There is an IO error preventing images to be saved.
[mapillary_trace (12).txt](https://github.com/mapillary/mapillary_issues/files/318199/mapillary_trace.12.txt)

**Comment by Caboosey** [comment]
I started seeing this since v2.25 and now seeing it more often now. I saw this error today after doing a single long sequence with over 5k photos and even started doing it after trying to start new sequences. 

However, the Mapillary app failed write to log file at same time and I cant find it in my logs.

This occurred on v2.29 running on Android 6.0.1 on a Samsung Galaxy S5 (Sprint) with a 64GB microSD card that had 12GB free at the time with the storage cutoff setting at 1% around 2016-06-24 around 06:00PM (US Central Time)

[mapillary.txt](https://github.com/mapillary/mapillary_issues/files/332834/mapillary.txt)
